aziotctl/aziotctl-common/src/set_log_level.rs

```diff
@@ -0,0 +1,57 @@
+use std::fs;
+use std::io::prelude::*;
+use std::process::Command;
+
+use anyhow::{Context, Result};
+
+use crate::{program_name, ServiceDefinition};
+
+#[allow(clippy::missing_errors_doc)]
+pub fn set_log_level(services: &[&ServiceDefinition], level: log::Level) -> Result<()> {
+    for service in services.iter().map(|s| s.service) {
+        write_log_level_file(service, level)?;
```

Comment:
Add a `.with_context(|| format!("could not write log level service override for {}", service))` before the `?`